source/_integrations/kostal_plenticore.markdown

```diff
@@ -113,6 +113,38 @@ The following sensors are available in the library:
 | Battery Charge from PV Year   | kWh  | Energy charged to the battery from the PV of the current year. |
 | Battery Charge from PV Total  | kWh  | Energy charged to the battery from the PV total. |
 
+<div class='note'>
+The inverter does not provide any data for the energy which is going to the grid directly.
+</div>
+
+#### Common Template Sensors
```

Comment:
```suggestion
#### Common template sensors
```
I realize there is heading style capitalization used elsewhere in this text but also in the docs. But we are trying to move to sentence-style capitalization. Following [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/capitalization).